### PR TITLE
Fix editorconfig

### PR DESCRIPTION
**Pull request by ghost**
- Add missing .editorconfig
- Use process.env.INIT_CWD defined by npm to determine the root path rather than to use the git system command.

### CODE REVIEW

File: packages/editorconfig/index.js [hunk 1]
```diff
@@ -3,8 +3,7 @@ const fs = require('fs')
 const path = require('path')
 const execSync = require('child_process').execSync
```
Comment (pa-bru) [inline packages/editorconfig/index.js]: you can remove this useless line

**Review by pa-bru — CHANGES_REQUESTED** [review]

File: packages/editorconfig/index.js [hunk 2]
```diff
@@ -3,8 +3,7 @@ const fs = require('fs')
...
 const source = path.join(__dirname, '.editorconfig')
-const gitRoot = execSync('git rev-parse --show-toplevel').toString().trim()
-const dest = path.join(gitRoot, '.editorconfig')
+const dest = path.join(process.env.INIT_CWD, '.editorconfig')
```
Comment (pa-bru) [inline packages/editorconfig/index.js]: are you sure it resolves the path you really want ? this env variable gives you the full path you were in when you ran npm run. it doesn't necessarily give you the top level directory path.

may you want to add the editorconfig file elsewhere than in the root directory ?

**Review by pa-bru — CHANGES_REQUESTED** [review]

File: packages/editorconfig/index.js (hunk 1 above)
Comment (ghost) [reply packages/editorconfig/index.js]: Done

File: packages/editorconfig/index.js (hunk 2 above)
Comment (ghost) [reply packages/editorconfig/index.js]: Can we run npm in another directory than the root directory ?

Comment (pa-bru) [reply packages/editorconfig/index.js]: yes

Comment (ghost) [reply packages/editorconfig/index.js]: J'ai revert ce que j'ai fait avec INIT_CWD

**Pull request closed without merging**